alpha/apps/kaltura/lib/myEntryUtils.class.php

```diff
@@ -807,10 +807,13 @@ public static function resizeEntryImage( entry $entry, $version , $width , $heig
 			if ($processing)
 				KExternalErrors::dieError(KExternalErrors::PROCESSING_CAPTURE_THUMBNAIL);
 
+			$videoRotation = mediaInfoPeer::retrieveOriginalByEntryId($entry->getId())->getVideoRotation();
```

Comment:
Export this code into a static function 
bool shoudRotate($entryId ,$vid slices)
And use it.